### ISSUE TITLE
Files get staged unnecessarily

### ISSUE THREAD

**Issue by gkotian** [gkotian/git_scripts#9]
If the `git_add_gui.sh` script is run, but all diffs are simply quit after being launched (i.e. no changes are actually added for staging), then it is observed that files still get staged.
Then on running `git diff --staged`, no changes are actually reported. This unnecessary staging should not happen at all.

**Comment by gkotian** [comment]
This is because of a mode change that happens. Need to investigate why.
```
diff --git a/initial_setup.sh b/initial_setup.sh
old mode 100755
new mode 100644
```

**Comment by gkotian** [comment]
Fixed by #17.
